src/test/resources/api/users/fakestoreapi/_post/rq.json

```diff
@@ -0,0 +1,12 @@
+[
+  {"id":1,
+    "title":"Fjallraven - Foldsack No. 1 Backpack, Fits 15 Laptops",
+    "price":2005.12,
+    "description":"Your perfect pack for everyday use and walks in the forest. Stash your laptop (up to 15 inches) in the padded sleeve, your everyday",
+    "category":"Body Builders Clothing",
+    "image":"https://fakestoreapi.com/img/81fPKd-2AYL._AC_SL1500_.jpg",
+    "rating":{
+      "rate":3.9,
+      "count":120}
+  }
+]
```

Comment:
Lets generate random values for every new product. check how to add properties file ([wildcards in the doc](https://zebrunner.github.io/carina/automation/api/#wildcards))